internal/corazawaf/transaction_test.go

```diff
@@ -429,6 +429,19 @@ func TestAuditLog(t *testing.T) {
 	}
 }
 
+func TestParseAuditLog(t *testing.T) {
+	AuditLogParts, err := types.ParseAuditLogParts("ABCDEFGHIJK")
+	if err != nil || string(AuditLogParts) != "ABCDEFGHIJK" {
```

Comment:
I think tinygo does not like this casting ` string(AuditLogParts)` because AuditLogParts is a named type. Could you compare length and byte by byte?